Assets/Scripts/CameraFollow.cs

```diff
@@ -43,14 +43,15 @@ private bool CheckYMargin()
 
     private void Update()
     {
-		if (Input.GetKey (KeyCode.C)) {
-			player.GetComponent<Controller2> ().enabled = false;
-			moveCamera ();
-		} else {
-			player.GetComponent<Controller2> ().enabled = true;
-			TrackPlayer();
-		}
-
+		if (!player.GetComponent<Controller2> ().ghostMode) {
+			if (Input.GetKey (KeyCode.C)) {
```

Comment:
I think we can change this once we agree on the controls, cos ur thinking of changing controls for ghostmode as well right